src/components/ExpandedViewMode/ExpandedViewModeContent/ActivityHistory/ActivityHistory.js

```diff
@@ -9,8 +9,11 @@ import StatusIcon from '../../../StatusIcon'
 class ActivityHistory extends Component {
   constructor(props) {
     super(props)
-    this.state = {}
+    this.state = {
+      timerId: 0,
+    }
     this.componentDidMount = this.componentDidMount.bind(this)
+    this.componentWillUnmount = this.componentWillUnmount.bind(this)
```

Comment:
You can remove these last two ^^ lines since these are built in lifecycle  methods.